vars/circleCiInstall.groovy

```diff
@@ -13,7 +13,8 @@
 
 def call() {
   withEnv(["DESTDIR=${env.WORKSPACE}/circleci/",
-           "PATH=${env.PATH}:${env.WORKSPACE}/circleci/"]) {
+           "PATH=${env.PATH}:${env.WORKSPACE}/circleci/",
+           "VERSION=0.1.5879"]) {
```

Comment:
Minor - if possible can this be made a constant so its clear what the version applies to?

Reply:
updated in d519023